optimize docs for event system (#1903)
cocos-creator/creator-docs: en/engine/event/event-input.md

# Input Event System

> As mentioned in the previous document, `EventTarget` supports a complete set of event listening and emitting mechanism.  

Reviewer: ```suggestion
As mentioned in the previous document, `EventTarget` supports a complete set of event listening and emitting mechanisms.  
```

> In Cocos Creator v3.4.0, we support the `input` object, which implements the event registering interface of `EventTarget`, and can register global system input events through this object. 

Reviewer: ```suggestion
Support for the `input` object, which implements the event registering interface of `EventTarget`, and can register global system input events through this object. 
```

Reviewer: I removed Creator v3.4 as extraneous

Author: the `input` is supported on since v3.4.0, should we specify the engine version ?

Author: I change in this way https://github.com/cocos-creator/creator-docs/pull/1903/commits/e42e2c163f86896c0365159c133300dd3f5b9de8

> > __Note__: The `systemEvent` object has been deprecated since v3.4.0. The difference with the `input` object is that the event listener of `systemEvent` will be intercepted by the node's event listener, while the `input` object has a higher priority than the node and will not be intercepted.

Reviewer: ```suggestion
> __Note__: the `systemEvent` object has been deprecated starting at v3.4.0. The difference with the `input` object is that the event listener of `systemEvent` will be intercepted by the node's event listener, while the `input` object has a higher priority than the node and will not be intercepted.
```

```

Please review the [test-cases-3d](https://github.com/cocos-creator/test-cases-3d/tree/v3.3/assets/cases/event) (This includes the keyboard, accelerometer, single point touch, multi-touch examples).

## Touch detection for 3D objects

The touch detection for 3D objects and 2D UI nodes is different. 2D UI nodes only need the size information provided by the `UITransform` component and the position information of the node to do the touch detection. For details, please refer to [Node Event System](event-node.ts). 

> The touch detection for 3D objects needs to be implemented by ray cast. The specific method is to generate a ray from the rendering camera of the 3D object to the screen coordinates of the touch point to determine whether the ray hits the object you want to detect. The specific code implementation is as follows: 

Reviewer: ```suggestion
The touch detection for 3D objects needs to be implemented by ray cast. The specific method is to generate a ray from the rendering camera of the 3D object to the screen coordinates of the touch point to determine whether the ray hits the object that was detected. The specific code implementation is as follows: 
```
